### PR TITLE
Header

### PR DESCRIPTION
**Pull request by sacidcelik**
Please review :)

Implemented a sticky header component that is rendered on all pages and should redirect to '/home' on click.

If you want to try this locally, starting the server via `npm run dev` from root should work if you remove the api key from the fetch function request in the backend (server/controller/search.controller.js):

`const searchAddress = 'https://www.googleapis.com/books/v1/volumes?key=${apiKey}&maxResults=25&q='; `-> `const searchAddress = 'https://www.googleapis.com/books/v1/volumes?maxResults=25&q=';`

(plus `npm install` in root, client and server folders :))

Thanks
Sacid

### CODE REVIEW

File: client/src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
```
Comment (Morten-Moeller) [inline client/src/components/Header.js:2]: I recommend using styled-components with macro. Then the rendered styles in the HTML got the component name attached.
```suggestion
import styled from 'styled-components/macro';
```

File: client/src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+const HeaderWrapper = styled.header`
+  align-items: flex-end;
+  background: var(--background);
+  border-bottom: 1px solid var(--secondary);
+  display: flex;
+  height: 70px;
+  justify-content: center;
+  margin: ${(props) => (props.isStatic ? 'auto' : '')};
+  margin-bottom: 1rem;
+  padding-bottom: 5px;
```
Comment (Morten-Moeller) [inline client/src/components/Header.js:30]: Why use pixel in this case?

**Review by Morten-Moeller — COMMENTED** [review]
Good job 👾
Keep on rolling!

File: client/src/components/Header.js (hunk 2 above)
Comment (sacidcelik) [reply client/src/components/Header.js:30]: Wanted to allow for pixel perfect styling taken from XD in this case. I could have converted it to REM though, true!

**Pull request merged**